docs/_sass/_layout.scss

```diff
@@ -118,4 +118,17 @@
 .highlight .n {
   color: #008000;
   font-weight: normal;
+}
+
+@media (min-width: 66.5rem) {
+  .side-bar {
+    width: calc((100% - 90rem) / 2 + 16.5rem);
+    min-width: 16.5rem;
+  }
+}
+
+@media (min-width: 66.5rem) {
```

Comment:
I'm not such a (s)css expert, but can't those two block be merged into one?

Reply:
Indeed, it is corrected